test/sablon_test.rb

```diff
@@ -150,12 +150,12 @@ def test_generate_document_from_template
     template = Sablon.template @template_path
     #
     # setup two image contents to allow quick reuse
-    r2d2 = Sablon.content(:image, @image_fixtures.join('r2d2.jpg').to_s)
-    c3po = Sablon.content(:image, @image_fixtures.join('c3po.jpg'))
-    darth = Sablon.content(:image, @image_fixtures.join('darth_vader.jpg'))
+    r2d2 = Sablon.content(:image, @image_fixtures.join('r2d2.jpg').to_s, properties: {unit: 'cm', height: '2', width: '2'})
```

Comment:
We probably don't need to set the dimensions for all the images just one or two to act as an integration test covering both the cases with and without dimensions.

Reply:
Makes sense 😃 - done